tools/flowc/backends/nim/runtime.nim

```diff
@@ -49,13 +49,10 @@ template rt_concat_strings*(s1: RtString, s2: RtString): RtString =
     s1 & s2
 
 template rt_string_len*(s: RtString): int32 =
-  when use16BitString: int32(s.len) else: int32(s.runeLen)
+  when use16BitString: int32(s.len) else: int32(s.len)
 
 template rt_string_char_code*(s: RtString, i: int32): int32 =
-  when use16BitString: int32(cast[uint16](s[i])) else: int32(s.runeAtPos(i))
-
-template rt_char_code_to_string*(code: int32): RtString =
-  when use16BitString: @[Utf16Char(cast[int16](code))] else: unicode.toUTF8(code)
+  when use16BitString: int32(cast[uint16](s[i])) else: int32(s.runeAt(i))
```

Comment:
When the rt_char_code_to_string is used to implement the flow getCharCodeAt then for "use16BitString" is it does look like it always return a string with one uint16. It it should sometime be two words (like in the Java native)